django_redis/compressors/gzip.py

```diff
@@ -0,0 +1,19 @@
+import gzip
+
+from django_redis.compressors.base import BaseCompressor
+from django_redis.exceptions import CompressorError
+
+
+class GzipCompressor(BaseCompressor):
+    min_length = 15
+
+    def compress(self, value: bytes) -> bytes:
+        if len(value) > self.min_length:
+            return gzip.compress(value)
+        return value
+
+    def decompress(self, value: bytes) -> bytes:
+        try:
+            return gzip.decompress(value)
+        except gzip.BadGzipFile as e:
```

Comment:
this is not available for python 3.6 and 3.7, read [here](https://docs.python.org/3.8/library/gzip.html#gzip.BadGzipFile)